src/report/report.ts

```diff
@@ -82,4 +108,19 @@ export class ReportModule {
     private isApiResponseError(error: ApiResponseError | Error): error is ApiResponseError {
         return (error as ApiResponseError).response !== undefined;
     }
+
```

Comment:
These two methods were reverted after they were removed in https://github.com/gooddata/gooddata-js/commit/bcb7d20a56b6af529c187f1067dbaec94f55daf0

Originally the method was used to sanitize just the filters that sent instead of AFM. I got the methods back and used them to sanitize filters inside of the AFM (and convert it from AFM to ExecuteAFM type).